### PR TITLE
Annotate New Examples from Recently Closed V7-Feature PRs

### PR DESCRIPTION
**Pull request by RexJaeschke**
I looked for all V7-feature PRs that were closed since I finished annotating all chapters back in January. I found 4: 240, 664, 239, and 244. However, only 240 and 664 added new examples, which I have annotated here.

As more V7-feature PRs get closed, I'll do likewise for them.

### CODE REVIEW

**Comment by jskeet** [comment]
I'll try to get to this next week - it won't be before then, I'm afraid.

**Review by BillWagner — APPROVED** [review]
These changes look fine to me @RexJaeschke 

I'll wait until @jskeet gets the chance to look before we merge.

**Review by jskeet — APPROVED** [review]
Yup, looks good to me, thanks Rex!

**Pull request merged**
